python_modules/dagit/dagit_tests/test_graphql.py

```diff
@@ -137,296 +141,115 @@ def test_pipelines():
 
 def test_pipeline_by_name():
     result = execute_dagster_graphql(
-        define_repo(), '''
-    { 
-        pipeline(name: "pandas_hello_world_two") { 
-            name 
+        define_repo(),
+        '''
+    {
+        pipeline(name: "pandas_hello_world_two") {
+            name
         }
-    }'''
+    }''',
     )
 
     assert result.data
     assert not result.errors
     assert result.data['pipeline']['name'] == 'pandas_hello_world_two'
 
 
-def test_production_query():
+def test_query_compute_nodes():
     result = execute_dagster_graphql(
-        define_repo(),
-        PRODUCTION_QUERY,
+        define_repo(), '''
+query PipelineQuery($config: String!)
+{
+  pipeline(name:"pandas_hello_world") {
+    name
+    computeNodeGraph(config:$config) {
+      computeNodes {
+        name
+        solid {
+            name
+        }
+        tag
+        inputs { 
+           name
+           type {
+               name
+           }
+           dependsOn {
+               name
+           }
+        }
+        outputs {
+            name
+            type {
+                name
+            }
+        }
+      }
+    }
+  }
+}
+        ''', {
+            'config':
+            '''
+solids:
+  load_num_csv:
+    config:
+      path: "pandas_hello_world/num.csv"
+
+            ''',
+        }
     )
 
-    if result.errors:
-        raise Exception(result.errors)
-
     assert result.data
     assert not result.errors
 
+    compute_node_graph_data = result.data['pipeline']['computeNodeGraph']
```

Comment:
Should we snapshot test here instead?

Reply:
ya can do that. at first i generally develop tests incrementally and want super granular errors.